docs/dev/table/tableApi.md

```diff
@@ -4490,6 +4490,16 @@ dateFormat(TIMESTAMP, STRING)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+timestampDiff(unit, timestamp1, timestamp2)
```

Comment:
timestampDiff(unit, timestamp1, timestamp2) => timestampDiff(TimeIntervalUnit, datetime1, datetime2)
TIMESTAMP is different from DATETIME. The supported range of DATETIME is '1000-01-01 00:00:00' to '9999-12-31 23:59:59', while TIMESTAMP has a range of '1970-01-01 00:00:01' UTC to '2038-01-19 03:14:07' UTC